core/src/impl/Kokkos_ExecSpaceManager.hpp

```diff
@@ -26,7 +26,7 @@
 #include <string>
 #include <utility>
 
-namespace {
+namespace Kokkos::Impl {
```

Comment:
Same here, I did mean to hide these symbols.

Reply:
If we do that then we'd include everything in that nested namespace instead of closing and then open Kokkos followed by open Impl.

Reply:
FWIW I could not see the symbols with your change